hive-metastore/src/test/java/org/apache/iceberg/hive/HiveCreateReplaceTableTest.java

```diff
@@ -197,42 +196,41 @@ public void testReplaceTableTxnTableModifiedConcurrently() {
 
     // the replace should still succeed
     table = catalog.loadTable(TABLE_IDENTIFIER);
-    Assert.assertNull("Table props should be updated", table.properties().get("another-prop"));
-    Assert.assertEquals("Table props should match", "value", table.properties().get("prop"));
+    Assertions.assertThat(table.properties().get("another-prop")).as("Table props should be updated").isNull();
+    Assertions.assertThat(table.properties().get("prop")).as("Table props should match").isEqualTo("value");
```

Comment:
both assertions can be combined to `assertThat(table.properties()).doesNotContainKey(...).containsEntry(..)`

Reply:
Addressed